agent/api/task.go

```diff
@@ -125,6 +125,10 @@ type Task struct {
 	// ENI is the elastic network interface specified by this task
 	ENI     *ENI
 	eniLock sync.RWMutex
+
+	// memoryCPULimitsEnabled to determine if task supports CPU, memory limits
+	memoryCPULimitsEnabled     bool
```

Comment:
Does this need to be saved in the state file?  Or are we recomputing when the task is read from the state file?

Reply:
Are you going to persist the task limit settings or cgroup spec here?  We need something that indicates to a restarted agent that a task has a parent cgroup and whether it has already been set up.

Reply:
We can persist this onto the state file but its still not going to indicate if the cgroup hierarchy has been setup for the task.